common/io/Serial.cpp

```diff
@@ -263,9 +262,6 @@ bool AcquireUUCPLockAndOpen(const std::string &path, int oflag, int *fd) {
   // further opens.
   if (ioctl(*fd, TIOCEXCL) == -1) {
     OLA_WARN << "TIOCEXCL " << path << " failed: " << strerror(errno);
-    close(*fd);
```

Comment:
Again we should close.

Reply:
It now closes (and unlocks, if necessary) on all exit paths.